key_action_map.cpp

```diff
@@ -310,4 +386,22 @@ void test_key_action_map()//!OCLINT tests can be many
     assert(kam.to_key(action_type::shoot_stun_rocket) == sf::Keyboard::E);
   }
 #endif // FIX_ISSUE_355
+
+//#define ISSUE_522
+#ifdef ISSUE_522
+  {
+    const key_action_map kam = get_player_1_kam();
+    std::stringstream s;
+    s << kam;
+  }
+  {
+    const key_action_map kam = get_player_1_kam();
+    const std::string filename = "test.txt";
+    save_to_file(kam, filename);
+    const key_action_map map_again = load_kam(filename);
+    assert(kam == map_again);
+    // TODO: delete temporary file
```

Comment:
Here we need another issue to remind us to delete this temporary file

Reply:
@richelbilderbeek I've reverted my last commit on your branch, I'll switch to my own branch, let's open an issue for this. But can we approve this PR without functioning tests?

Reply:
@EvoLandEco yes, one can approve of the tests :-) . Good tests are hard to write :+1: